### PR TITLE
Added Spanish translations

### PR DESCRIPTION
**Pull request by DJNeonNight**
(Hopefully everything is right)

### CODE REVIEW

File: src/js/translations.js [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
   en,
   de,
+  es,
 };
 
 export const languages = {
   'en': 'English',
   'fi': 'Suomi (Finnish)',
   'de': 'Deutsch (German)',
+  'en': 'Spanish (Spain)',
```
Comment (oamaok) [inline src/js/translations.js]: this should be `Español (Spanish)`

File: src/translations/es.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "error-occured": "Un error ha impedido que la extensión funcione apropiadamente: ",
+  "version-update-message": "¡Ahora estás usando ezpp!",
```
Comment (oamaok) [inline src/translations/es.json]: just to make sure, this is used in the context of version changes as in `You are now using ezpp! v.1.5.0`, which shows up when the extension is updated

Comment (oamaok) [reply src/translations/es.json]: i should probs make it a template string to make it more clear

File: src/js/translations.js (hunk 1 above)
Comment (DJNeonNight) [reply src/js/translations.js]: I see, so "Deutsch (German)" should not be in English either, right?

File: src/translations/es.json (hunk 2 above)
Comment (DJNeonNight) [reply src/translations/es.json]: Alright, I'll change it a bit so it looks better, the ¡ ! definitively look weird if the version will be after them

File: src/js/translations.js (hunk 1 above)
Comment (oamaok) [reply src/js/translations.js]: `Deutsch` is German for German, so it's correct. the format is supposed to be `name of the original language (name of the language in english)`

**Comment by oamaok** [comment]
Thank you!

**Pull request merged**